### PR TITLE
docs - sankey

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Adds sankey chart to list of charts.
- Alphabetizes charts in docs list.

### CODE REVIEW

File: docs/questions/sharing/visualizations/sankey.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+| Homepage     | Exit         | 810            |
+| Product Page | Checkout     | 815            |
+| Checkout     | Purchase     | 2,217          |
+| Homepage     | Exit         | 1,020          |
+
+See [data options](#sankey-data-options).
+
+### Circular dependencies won't work
+
+If some of your sources point to targets that point back to the same sources, Metabase won't be able to create a sankey chart.
```
Comment (alexyarosh) [inline docs/questions/sharing/visualizations/sankey.md:42]:
```suggestion
If some of your sources point to targets that point back to the same sources, Metabase won't be able to create a Sankey chart.
```

Sankey is name

File: docs/questions/sharing/visualizations/sankey.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+You can select left, right, or justified alignment for the sankey chart. Alignment determines how the chart should display the end nodes (a.k.a. leaf nodes or terminal nodes).
+
+The chart in the section above is left-aligned. The end nodes, `Exit` and `Purchase`, are aligned to the left.
+
+For right alignment, the end nodes, `Exit` and `Purchase`, move to the chart's right:
+
+![Right-aligned sankey chart](../../images/sankey-right-aligned.png)
+
+In this case, justified alignment looks the same, as the end nodes move to take up the whole chart.
```
Comment (alexyarosh) [inline docs/questions/sharing/visualizations/sankey.md:66]: What does justified alignment do in general? Is it not always just making a right-aligned chart?

File: docs/questions/sharing/visualizations/sankey.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+## Sankey data shape
+
+To create a Sankey chart, you'll need at least three columns in your results:
+
+- **Source column**: specifies a node in the sankey flow.
+- **Target column**: specifies a receiving node.
+- **Count column**: value that determines the thickness of the target node.
+
+Here's the data shape used to create the chart above.
```
Comment (alexyarosh) [inline docs/questions/sharing/visualizations/sankey.md:18]: This is great as an example but tbh I'm really confused how to build a QB question that would give a result like this

File: docs/questions/sharing/visualizations/sankey.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+See [data options](#sankey-data-options).
+
+### Circular dependencies won't work
+
+If some of your sources point to targets that point back to the same sources, Metabase won't be able to create a sankey chart.
+
+## Sankey data options
+
+To select which columns Metabase should use as the source, target, and values for the sankey chart, click on the **settings** icon in the bottom right to view the **Data** tab.
```
Comment (alexyarosh) [inline docs/questions/sharing/visualizations/sankey.md:46]:
```suggestion
To select which columns Metabase should use as the source, target, and values for the Sankey chart, click on the **settings** icon in the bottom right to view the **Data** tab.
```

File: docs/questions/sharing/visualizations/sankey.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+Your data can include more than three columns, but each row must include the required three columns: source, target, and a value to scale the size of the target node.
+
+## Sankey display options
+
+You can change a charts alignment, edge labels, and edge colors.
+
+### Alignment
+
+You can select left, right, or justified alignment for the sankey chart. Alignment determines how the chart should display the end nodes (a.k.a. leaf nodes or terminal nodes).
```
Comment (alexyarosh) [inline docs/questions/sharing/visualizations/sankey.md:58]:
```suggestion
You can select left, right, or justified alignment for the Sankey chart. Alignment determines how the chart should display the end nodes (a.k.a. leaf nodes or terminal nodes).
```

File: docs/questions/sharing/visualizations/sankey.md (hunk 2 above)
Comment (alexyarosh) [reply docs/questions/sharing/visualizations/sankey.md:66]: Answer: It aligns all entry nodes to the left and all end nodes to the right

**Review by alexyarosh — APPROVED** [review]

File: docs/questions/sharing/visualizing-results.md [hunk 6]
```diff
@@ -27,87 +27,109 @@ To change the settings for a specific chart, for example a row chart, you could
...
+
+The [Numbers](./visualizations/numbers.md) option is for displaying a single number, nice and big.
+
+![Number](../images/number.png)
+
+## Pie, donut, and sunburst charts
+
+A [pie chart or donut chart](./visualizations/pie-or-donut-chart.md) can be used when breaking out a metric by a single dimension, especially when the number of possible breakouts is small, like accounts by plan.
+
+A [sunburst chart](./visualizations/pie-or-donut-chart.md) chart is a pie chart with more than one ring to show the data broken out by additional dimensions.
```
Comment (abhishek-superk) [inline docs/questions/sharing/visualizing-results.md:88]: You're calling it "A sunburst chart chart is a pie chart", should remove the second chart.

**Pull request merged**